js/Model.js

```diff
@@ -7,17 +7,17 @@ define(
 		'services/CohortDefinition',
 		'services/ConceptSet',
 		'services/role',
-		'providers/AutoBind',
-		'providers/Vocabulary',
+		'utils/AutoBind',
+		'services/Vocabulary',
```

Comment:
I'm seeing the Vocabulary service as 'services/Vocabulary' but later (I will mark it below) i see a 'services/VocabularyProvider'.  Is this something that should be merged into a single VocabularyAPI or VocabularyService?